TEST

```diff
@@ -85,6 +85,20 @@ esac
 
 ###
 #
+# Run ztest longer on the "Coverage" builders to gain more code coverage
+# data out of ztest, libzpool, etc.
+#
+case "$BB_NAME" in
+*Coverage*)
```

Comment:
The `c` needs to be lower case I believe. Slave names for the Coverage builder are something like `Ubuntu-17.04-x86_64-coverage-testslave1`.

Reply:
Thank you! I noticed it didn't run longer and was confused. Are the names documented anywhere? I was just looking at the buildbot UI and the github status and thought it would have been: `Ubuntu 17.04 x86_64 Coverage (TEST)`

Reply:
So buildbot has a notion of a Builder (that is `Ubuntu 17.04 x86_64 Coverage (TEST)` in this case). These Builders have Build Slaves (in this case `Ubuntu-17.04-x86_64-coverage-testslave1`). Both Builders and Build Slaves get assigned unique names. `BB_NAME` within our buildbot scripts is actually the Build Slave's name.

These are documented in `master.cfg` in the `zfs-buildbot` repo. You'll see sets of Build Slaves being instantiated and assigned to builders in there. The coverage Build Slaves are instantiated [here](https://github.com/zfsonlinux/zfs-buildbot/blob/d2c2abf76bc87ac9bd0b3d39f0887b9aa8cfcd24/master/master.cfg#L783). The Builders are then assigned those slaves [here](https://github.com/zfsonlinux/zfs-buildbot/blob/d2c2abf76bc87ac9bd0b3d39f0887b9aa8cfcd24/master/master.cfg#L1235).

Feel free to post more questions here or contact me on Slack.